README.md

```diff
@@ -1,2 +1,21 @@
+![logo](./src/files/assets/seneca-logo.png)
+> A Node.js toolkit for micro service architectures
+
+[![Gitter][gitter-badge]][gitter-url]
+
 # getting-started
 Code examples for the Getting Started Guide
```

Comment:
Change this to:

This repo contains all of the code examples from the [Getting started guide][] in [Sencajs.org][]